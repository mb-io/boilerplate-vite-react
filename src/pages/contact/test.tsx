import { render } from '@testing-library/react'
import ContactPage from './contact'
import { describe, it, expect } from 'vitest'

describe('ContactPage', () => {
  it('renders correctly', () => {
    const { container } = render(<ContactPage />)
    expect(container).toMatchSnapshot()
  })
})
