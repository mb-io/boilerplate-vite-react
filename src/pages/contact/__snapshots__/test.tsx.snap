// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ContactPage > renders correctly 1`] = `
<div>
  <section
    class="flex flex-col items-center justify-center gap-4 py-8 md:py-10"
  >
    <div
      class="inline-block max-w-lg justify-center text-center"
    >
      <h1
        class="inline font-semibold tracking-tight text-[2.3rem] leading-9 lg:text-5xl"
      >
        Contact
      </h1>
      <div
        class="my-2 block w-full max-w-full text-lg text-default-600 md:w-1/2 lg:text-xl !w-full mt-4"
      >
        Coming soon...
      </div>
    </div>
  </section>
</div>
`;
